virttest/qemu_devices/qcontainer.py

```diff
@@ -1826,7 +1826,7 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
                 elif scsi_hba == 'virtio-scsi-ccw':
                     pci_bus = None
             elif scsi_hba == 'spapr-vscsi':
-                addr_spec = [8, 16384]
+                addr_spec = [64, 32]
```

Comment:
Here qemu allows 0-255 and 0-16383. I'll bring this on s390x meeting to see whether it's expected

Reply:
(the meeting is in ~2h and I forgot to mention I used about a month old upstream qemu)

Reply:
OK so I used IRC and zkvm team would like to use the current limit, which is `[256, 16384]`. Would you please update this? (unless you have some good reason not to...)